src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -14,17 +14,23 @@ fi
 
 script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 app_dir="$( realpath -e "${script_dir}/build")"
+service_name=$(getConfItem service_name)
+
 cf create-org test
 cf target -o test
 cf create-space test_app
 cf target -s test_app
+
+cf enable-service-access "$service_name" -b "$(getConfItem service_broker)" -p  "$(getConfItem service_plan)" -o test
+cf create-service "$service_name" "$(getConfItem service_plan)" "$service_name" -b "$(getConfItem service_broker)" --wait
+#cf unbind-service test_app "$service_name" --wait
+
 pushd "$app_dir" >/dev/null
 cf push\
   --var app_name=test_app\
   --var app_domain="$(getConfItem apps_domain)"\
-  --var service_name="$(getConfItem service_name)"\
+  --var service_name="$service_name"\
```

Comment:
It is not required (at least not here), but I personally like the usage of the curly braces for each variable reference. It makes things less ambiguous. Be it with parameter extension or with the suprising similariy of `$` and `S` in the font I currently see.